### DISCUSSION TITLE
More languagues

### DISCUSSION THREAD

**Discussion by iVillager** [GriefPrevention/GriefPrevention#1291] [Ideas]
Can you add more languages to the plugin? Also opening a page where people can help by translating would be awesome!

**Comment by leadvic** [comment]
Yeah, I'd love to help translate this plugin to spanish!

**Reply by piodois** [reply]
lo conseguiste?

**Reply by thiagorigonatti** [reply]
> Yeah, I'd love to help translate this plugin to spanish!

https://github.com/GriefPrevention/GriefPrevention/discussions/705#discussioncomment-17268466

**Comment by thiagorigonatti** [comment]
> Can you add more languages to the plugin? Also opening a page where people can help by translating would be awesome!

https://github.com/GriefPrevention/GriefPrevention/discussions/705#discussioncomment-17268466
